app/models/ems_refresh/save_inventory.rb

```diff
@@ -101,7 +102,11 @@ def save_vms_inventory(ems, hashes, target = nil)
         found.raw_power_state = key_backup[:raw_power_state]
 
         link_habtm(found, key_backup[:storages], :storages, Storage)
-        link_habtm(found, key_backup[:security_groups], :security_groups, SecurityGroup)
+        # TODO(lsmola) NetworkManager, once all providers are converted, security groups relation of Vm will go away,
+        # being moved to NetworkPort, we will ignore here through associations
+        if found.try(:security_groups).try(:proxy_association).try(:reflection).try(:options).try(:[], :through) != :network_ports
```

Comment:
@Fryguy, @matthewd do you guys have any suggestions around the chained `try`s here?

Looks like this is basically temporary until we can get the full realization of the network provider.  So, maybe it doesn't need to be solved.  But, just wondering if there's a better way.